README.md

```diff
@@ -18,33 +18,37 @@ BLAST.
 Given a sequence database (in FASTA format), `vpsearch build` constructs an
 optimized vantage point search tree. Building the tree is a one-time operation
 and doesn't have to be done again unless the database changes. As an
-illustration, we build a vantage point tree for the RDP database of bacterial
-16S sequences. This database contains 281261 sequences of which 39237 are
-duplicates. After removing these duplicates, we are left with 242024 unique
-sequences. Building a tree for these sequences is done with:
-```bash
-  $ vpsearch build rdp_download_281261seqs_dedup.fa
-  Building for 242024 sequences...
-  done.
+illustration, we build a vantage point tree for a database of sequences
+obtained by trimming the GTDB 16S database to the v3-v4 hypervariable
+region. This database contains 10875 unique sequences, and can be found (in
+compressed form) in the `data/` directory inside this repository.
+```console
+  $ vpsearch build bac120_ssu_reps_r207-sliced-dedup.fa
+  Building for 10875 sequences...done.
   Linearizing...done.
-  Database created in rdp_download_281261seqs_dedup.db
+  Database created in bac120_ssu_reps_r207-sliced-dedup.db
 ```
-For the RDP database of full length sequences, this takes about 20 minutes on a
-standard machine. When only selected regions of the sequences are considered,
-the time needed to build a tree can be much reduced. For example, vantage point
-trees for the v1-v2 hypervariable region (350 base pairs) or the v3-v4 region
-(250 base pairs) of the RDP 16S sequencese can be built in 30 seconds to 1
-minute.
 
-Once a tree has been built, unknown sequences can be looked up using the
-`vpsearch query` command. Here we supply a query file with a single sequence
-```bash
-  vpsearch query rdp_download_281261seqs_dedup.fa query.fa
-  query	S000143715	99.54	1529	0	0	1	1524	1	1529	0	7546
-  query	S004085923	99.08	1529	0	0	1	1524	1	1526	0	7481
-  query	S004085922	99.08	1529	0	0	1	1524	1	1526	0	7481
-  query	S004085925	98.50	1531	0	0	1	1524	1	1527	0	7386
+As this is a relatively small database, the process finishes quickly, in about
+10 seconds. For larger databases, such as the RDP database of full length
+sequences, this may take longer. For example, building an index for the RDP
+database takes about 20 minutes on a standard machine.
 
+Once a tree has been built, unknown sequences can be looked up using the
+`vpsearch query` command. Here we supply a query file with a single
+sequence. The `query.fa` file can also be found in the `data/` directory and
+represents a Lactobacillus helsingborgensis sample whose sequence was
```

Comment:
In the interest of pedantry, genera and species should probably be italicized here and at L44.

Reply:
Thanks, done.